cmd/create/admin/cmd.go

```diff
@@ -24,13 +24,15 @@ import (
 	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
 	"github.com/spf13/cobra"
 
-	"github.com/openshift/rosa/cmd/create/idp"
 	"github.com/openshift/rosa/pkg/object"
 	"github.com/openshift/rosa/pkg/ocm"
 	"github.com/openshift/rosa/pkg/output"
 	"github.com/openshift/rosa/pkg/rosa"
 )
 
+const ClusterAdminUsername = "cluster-admin"
+const ClusterAdminIDPname = "cluster-admin"
```

Comment:
Do we need the two consts for the same value?

Reply:
@ciaranRoche ,  I do think we need two separate constants to distinguish between user-name and idp-name for clarity/readability of the code.  

It just so happened that in this case , we are choosing the same  name for  user as well as the idp holding the user.

Reply:
cool cool cool